lib/version.c

```diff
@@ -274,7 +274,7 @@ static const char * const protocols[] = {
 #ifndef CURL_DISABLE_RTSP
   "rtsp",
 #endif
-#if defined(USE_LIBSSH) || defined(USE_LIBSSH2)
+#if defined(USE_SSH)
```

Comment:
`lib/version.c` needs this diff too doesn't it?:
```patch
diff --git a/lib/version.c b/lib/version.c
index 403b04d5a..7a993b9e2 100644
--- a/lib/version.c
+++ b/lib/version.c
@@ -390,7 +390,7 @@ static curl_version_info_data version_info = {
 curl_version_info_data *curl_version_info(CURLversion stamp)
 {
   static bool initialized;
-#if defined(USE_LIBSSH) || defined(USE_LIBSSH2)
+#ifdef USE_SSH
   static char ssh_buffer[80];
 #endif
 #ifdef USE_SSL
```

Reply:
indeed. Thanks!